notebook/gen_ai_jss.ipynb

```diff
@@ -0,0 +1,670 @@
+{
```

Comment:
<p>I didn't realize that we had planned on deployment through a script. If that is still planned, we can still use deploy.sh, correct?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-genai-doc-summarization/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>